content/ADR-125-realm-modifiers-for-worlds-in-renderer.md

```diff
@@ -0,0 +1,191 @@
+    {
+        void OnEnteredRealm(bool isCatalyst, AboutResponse realmConfiguration);
+    }
+}
+```
+
+ The current modifiers currently implemented area:
+
+- **Realms Blocker Modifier**
+
+Controls how the green blockers should be shown on each realm. If a realm is a Catalyst, or it has a value on Configurations.CityLoaderContentServer; then the green blockers will be shown.
+
+- **Realms Minimap Modifier**
+
+Controls the functionality associated to the minimap when entering a Realm. It determines if the minimap should be visible using the realmConfiguration.Configurations.Minimap.Enabled value. Also, it determines if the Jump To Home Button should be present. This button will appear on any given user world, and it will take the user back to the home location in the most populated catalyst. 
+When needed, this modifier will implement the change of the minimap image; as a user generated world will have the possibility to show its own minimap png.
+	
+
+### Data Structure 
+
+`AboutResponse` contains the info of the realm which has just been accessed. This class is autogenerated according to protobuff specifications. Each modifier gets access to the AboutResponse, and gets the info necessary to execute its behaviour. It has the following structure: 
+
+```typescript
+type About = {
+  healthy: boolean
+  configurations: {
+    networkId: number
+    realmName?: string
+    scenesUrn: string[]
+    globalScenesUrn: string[]
+  }
+  content: {
+    healthy: boolean
+    version?: string
+    commitHash?: string
+    publicUrl: string
+  }
+  comms: {
+    healthy: boolean
+    protocol: string
+    fixedAdapter?: string
+    usersCount?: number
+  }
+  lambdas: {
+    healthy: boolean
+    version?: string
+    commitHash?: string
+    publicUrl: string
+  }
+  bff?: {
+    healthy: boolean
+    userCount: number
+    commitHash?: string
+    publicUrl: string
+  }
+}
+```
+
+Example:
+
+```json
+{
+  "healthy": true,
+  "content": {
+    "healthy": true,
+    "version": "5.1.2",
+    "commitHash": "dce61c002c89db4966c0cdd008d3d654f297050b",
+    "publicUrl": "https://peer.decentraland.org/content"
+  },
+  "configurations": {
+    "realmName": "catalyst-4",
+    "networkId": 1
+  },
+  "comms": {
+    "healthy": true,
+    "protocol": "v3",
+    "fixedAdapter": "ws-room:mini-comms.decentraland.org/rooms/test-room"
+  },
+  "lambdas": {
+    "healthy": true,
+    "version": "5.1.2",
+    "commitHash": "dce61c002c89db4966c0cdd008d3d654f297050b",
+    "publicUrl": "https://peer.decentraland.org/lambdas"
+  },
+  "bff": {
+    "healthy": true,
+    "commitHash": "369c5dafeda62a1b16f5232cd477565cc3f3d513",
+    "userCount": 0,
+    "publicUrl": "/"
+  }
+}
+```
+
+## Current Implementation 
+
+![Untitled Diagram drawio](https://user-images.githubusercontent.com/1999557/202185605-65ebf318-a2bd-40c9-8908-200daa2166b5.png)
+
+
+## Task breakdown 
+
+To complete the Beta version for worlds, the following modifiers are required:
```

Comment:
Can you specify that it's part of Explorer task breakdown

Reply:
Done!